infra/src/main/java/com/redhat/developer/infra/utils/exceptions/CloudEventDeserializationException.java

```diff
@@ -0,0 +1,12 @@
+package com.redhat.developer.infra.utils.exceptions;
+
+public class CloudEventDeserializationException extends RuntimeException {
```

Comment:
I wonder if this should extend `Exception` rather than `RuntimeException` otherwise we'll probably forget to catch the `RuntimeException` when we use it elsewhere in the code?

Reply:
See comment above